nat-lab/tests/uniffi/libtelio_proxy.py

```diff
@@ -56,7 +56,7 @@ def shutdown(self, container_or_vm_name: Optional[str] = None):
                 "Unknown" if container_or_vm_name is None else container_or_vm_name,
             )
 
-        except Pyro5.errors.ConnectionClosedError as e:
+        except (Pyro5.errors.ConnectionClosedError, Pyro5.errros.ConnectionRefusedError) as e:
```

Comment:
Does this really work with `||` or will it expect to catch tuple

Reply:
Seems like correct syntax: https://docs.python.org/3/tutorial/errors.html#handling-exceptions